src/applications/representative-search/containers/SearchPage.jsx

```diff
@@ -13,7 +13,7 @@ import appendQuery from 'append-query';
 import { browserHistory } from 'react-router';
 import repStatusLoader from 'applications/static-pages/representative-status';
 import environment from '@department-of-veterans-affairs/platform-utilities/environment';
-import { useFeatureToggle } from '~/platform/utilities/feature-toggles/useFeatureToggle';
+// import { useFeatureToggle } from '~/platform/utilities/feature-toggles/useFeatureToggle';
```

Comment:
Did you want to keep some of these lines commented out?

Reply:
Removed. Find a Rep won't load on older versions of Safari, and I have a hunch it's related to an issue with loading feature toggles. So I may bring the toggle back, but I'm good with deleting everything for now.